core/controllers/editor.py

```diff
@@ -67,6 +67,14 @@
     'in the gallery',
     default_value=MODERATOR_REQUEST_FORUM_URL_DEFAULT_VALUE)
 
+SHARING_OPTIONS_TWITTER_TEXT = config_domain.ConfigProperty(
+    'sharing_options_twitter_text_editor', {
+        'type': 'unicode',
+    },
+    'Default text for the Twitter share message for the editor',
+    default_value=(
+        'Check out this interactive lesson from Oppia - a free, open-source '
```

Comment:
Check out this interactive lesson I created on Oppia - a free platform for teaching and learning!